fw/compile-firmware.md

```diff
@@ -69,7 +72,7 @@ qmk config user.overlay_dir="$(realpath .)"
 
 ## Github actions {#actions-requirements}
 
-If you are want to use Github actions to compile your firmware (rather than the console), you will need to:
+If you want to use Github actions to compile your firmware (rather than the console), you will need to:
```

Comment:
```suggestion
If you want to use Github actions to compile your firmware (rather than doing it locally in the console), you will need to:
```
Not thrilled with this wording, but the differentiation should make clear that the alternative to GH actions is *local* compilation.